### PR TITLE
policy: update CORE_EXT pre-activation rc3 (#1342)

### LINKED ISSUE #1342: [Q-POLICY-CORE-EXT-PREACTIVATION-RC3-01] Replace CORE_EXT pre-activation policy with rc3 version
## Task contract

- queue_row_or_issue_id: `Q-POLICY-CORE-EXT-PREACTIVATION-RC3-01`
- target_repo: `rubin-protocol`
- one_invariant: the existing `POLICY_CORE_EXT_PREACTIVATION.md` is replaced with the rc3 pre-activation guardrail policy while preserving all governance and traceability anchors.
- architecture_class: `D = protocol policy document replacement`
- system_boundary: existing root policy document only; no code, no canonical spec changes.
- allowed_files_or_surfaces:
  - `POLICY_CORE_EXT_PREACTIVATION.md`
  - root policy index only if required
- non_goals: no new `_ENFORCEMENT` sibling file; no Go/Rust implementation; no activation semantics change; no CORE_EXT cost change.

## Accepted cases

- Preserve anchors: `COUNCIL_CORE_EXT_FRAMEWORK_20260315`, `COUNCIL_NATIVE_ROTATION_ARCH_20260315`, `Q-IMPL-CORE-EXT-COST-01`, `EXT_BASE_COST = 64`, `RUBIN_CORE_EXT_EXTENSION_FRAMEWORK.md`, CANONICAL §23.2.2 / §12.5 / §14.
- Filename remains `POLICY_CORE_EXT_PREACTIVATION.md`.
- Policy remains fail-closed pre-activation guardrail.

## Rejected cases

- Creating `POLICY_CORE_EXT_PREACTIVATION_ENFORCEMENT.md`.
- Dropping governance anchors.
- Changing activation or consensus semantics.
- Adding test override behavior outside documented unsafe test boundary.

## False-positive cases

- Content replacement is allowed if filename and anchors are preserved.
- Existing implementation issues remain separate.

## Hostile review matrix

- Anchor preservation grep must pass.
- No sibling duplicate policy file.
- No changed CORE_EXT base cost.
- No claims that runtime implementation changed.
- Cross-refs to spec paths must resolve or be explicitly marked external.

## Planned execution config

Protocol doc replacement PR; compare old policy file against rc3 version and preserve anchors.

## Mandatory local gates

- `git diff --check`
- grep all required anchors after edit
- link/path checks where local paths exist
- reviewer-first semantic pass for activation overclaim

## Evidence expected

- PR body lists before/after anchor grep results.
- PR states no implementation and no consensus change.

## Finding source

- rc3 archive file: `POLICY_CORE_EXT_PREACTIVATION.md`
- rc3 `MERGE_PLAN.md` P-A resolution.

## Dedup / why this is not existing work

No current open issue was found for rc3 replacement of this existing policy file.

## Class-change stop rule

If activation semantics, cost semantics, Go/Rust code, or canonical spec changes become necessary, stop and split.

## Follow-up policy

Implementation remains under existing CORE_EXT implementation tasks; this issue is policy-document alignment only.

## Archive provenance / JSON linkage

- rc3 pack: `rubin_policy_pack_postreview_rc3_2026-04-28`
- `DECISION_RECORD.json`: sha256 `50458b1cb7a0b4b73294d8a9b149102d58c0e857d22f62362ada367a947c16c4`, bytes `1834`
- `MANIFEST.json`: sha256 `d3a02883c40b9cfc419ee7da34f4e60234a0b3b17f3da10b2044780b048d87a7`, bytes `1947` (manifest file itself is not self-listed in `MANIFEST.json`)
- `VERIFICATION_REPORT.md`: sha256 `d329b8cd73947ac813e8dab124b8ba8a4a3210fe84173e07c5e9050980dcc272`, bytes `2191`
- `COMMENTARY.md`: sha256 `b49564b1506cc1f75e77f35a8f9938fcee87075f5343e6afc0038c430c4b5d26`, bytes `4229`

Relevant rc3 source file:
- `POLICY_CORE_EXT_PREACTIVATION.md`: sha256 `75a71b01873dc3032445d0fdcc009f31f13e646defe6cf954dc8115121e0fc78`, bytes `4627`

`DECISION_RECORD.json` / `MERGE_PLAN.md` requirements covered here:
- do not add a parallel CORE_EXT policy file;
- merge into existing filename `POLICY_CORE_EXT_PREACTIVATION.md`;
- preserve existing CORE_EXT governance anchors and parent references.

### PR DESCRIPTION
**Pull request by 2tbmz9y2xt-lang**
Closes #1342

Architecture class: D

Scope:
- Replace the existing root `POLICY_CORE_EXT_PREACTIVATION.md` with the rc3 CORE_EXT pre-activation policy.
- Preserve the existing filename and governance traceability anchors.
- Keep this PR documentation/policy-only.

Consensus impact: none.

No Go/Rust/conformance implementation.

Existing filename preserved.

Anchor grep passed.

Non-goals:
- No activation semantics change.
- No CORE_EXT cost change.
- No new sibling policy file.

### CODE REVIEW

File: POLICY_CORE_EXT_PREACTIVATION.md [hunk 1]
```diff
@@ -1,65 +1,218 @@
...
+## 2.1 Current Implementation Status
+
+This policy replacement does not claim new implementation coverage. It records policy requirements for
+wallet, RPC, mempool, relay, miner, unsafe-override, and telemetry surfaces when those surfaces implement
+CORE_EXT pre-activation handling.
+
+The currently documented implemented guardrail remains **miner template filtering**:
 
 - By default, `Miner` excludes any transaction that creates a `CORE_EXT` output or spends a `CORE_EXT` UTXO.
 - This is policy-only and does not change consensus validity.
```
[the remark this answers is not in the document]
Comment (2tbmz9y2xt-lang) [reply POLICY_CORE_EXT_PREACTIVATION.md]: Fixed in head d8c271f. §2.1 now states the current Go guardrail coverage as miner template filtering plus canonical mempool admission and P2P-relayed transaction admission through CanonicalMempoolTxPool, while keeping wallet/RPC/unsafe override/telemetry and non-canonical relay behavior out of scope.

File: POLICY_CORE_EXT_PREACTIVATION.md [hunk 2]
```diff
@@ -1,65 +1,218 @@
...
-The currently implemented guardrail is **miner template filtering**:
+This policy preserves the existing governance traceability surface. Existing anchors to retain in the repository record:
+
+```text
+COUNCIL_CORE_EXT_FRAMEWORK_20260315
+COUNCIL_NATIVE_ROTATION_ARCH_20260315
+Q-IMPL-CORE-EXT-COST-01
+EXT_BASE_COST = 64
+rubin-spec-private/spec/RUBIN_CORE_EXT_EXTENSION_FRAMEWORK.md
+rubin-core-ext-lab/RUBIN_CORE_EXT_EXTENSION_FRAMEWORK.md
```
[the remark this answers is not in the document]
Comment (2tbmz9y2xt-lang) [reply POLICY_CORE_EXT_PREACTIVATION.md]: Fixed in head d8c271f. The anchor now uses the repo convention `spec/RUBIN_CORE_EXT_EXTENSION_FRAMEWORK.md` instead of a local checkout path.

File: POLICY_CORE_EXT_PREACTIVATION.md [hunk 3]
```diff
@@ -1,65 +1,218 @@
...
-compliance profiles only. It MUST NOT be used as a native cryptographic suite
-replacement mechanism (COUNCIL_NATIVE_ROTATION_ARCH_20260315). Native suite
-lifecycle is governed by `RUBIN_NATIVE_CRYPTO_ROTATION_SPEC_v1.md`.
+```text
+core_ext_preactivation_reject_total
+core_ext_preactivation_miner_exclude_total
+core_ext_profile_active_total
+core_ext_profile_inactive_total
+core_ext_unknown_ext_id_total
+core_ext_unsafe_override_enabled
```
[the remark this answers is not in the document]
Comment (2tbmz9y2xt-lang) [reply POLICY_CORE_EXT_PREACTIVATION.md]: Fixed in head d8c271f. §9 now uses `rubin_node_core_ext_*` target metric names and explicitly says these are target names, not currently emitted metrics.
